deploifai/api/__init__.py

```diff
@@ -328,6 +328,36 @@ def get_projects(self, workspace, fragment: str):
         except KeyError as err:
             raise DeploifaiAPIError("Could not get projects. Please try again.")
 
+    def get_workspace(self, project_id: str, fragment: str):
+        query = (
+                """    
+                query($id:String) {
+                  projects(whereID: $id) {
+                    ...project
+                  }
+                }
+                """
+                + fragment
+        )
+
+        variables = {
+            "whereID": {"project_id": project_id},
+        }
```

Comment:
I believe the query is written a bit wrongly. The arguments are not correct, should check the graphql docs to see the expected type.

The name of the api function should be `get_projects` not `get_workspace`